src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3540,6 +3540,34 @@
     public static final int COMMENT_CONTENT =
             GeneratedJavaTokenTypes.COMMENT_CONTENT;
 
+    /**
+     * Java 14 Pattern definition.
```

Comment:
First sentence of javadoc is important. I would want to see what this token actually is than to know it is for Java 14.

Reply:
Fixed.